recipes-kernel/linux/4.9/patches/blktap2.patch

```diff
@@ -810,28 +810,13 @@ Index: linux-4.9.40/drivers/block/blktap/device.c
 +		goto fail;
 +	}
 +
-+	switch (req_op(rq)) {
-+		case REQ_OP_DISCARD:
```

Comment:
Are REQ_OP_DISCARD and REQ_OP_FLUSH no longer implemented?

Reply:
Yes, this was creating problems for blktap3. Need to be checked if these functionalities are required.

Reply:
The fact you were seeing problems without removing this piece is troubling.
This seems wrong as `blktap_ring_submit_request()` is not changed and is therefor inconsistent with this.

Could you describe the problems you were having?